lib/InputValidator.ts

```diff
@@ -2,11 +2,19 @@ import Encoder from './Encoder';
 import ErrorCode from './ErrorCode';
 import IonError from './IonError';
 import IonPublicKeyPurpose from './enums/IonPublicKeyPurpose';
+import Multihash from './Multihash';
 
 /**
  * Class containing input validation methods.
  */
 export default class InputValidator {
+  /**
+   * Validates that the given input is a multihash computed using a configured hash algorithm.
+   */
+  public static validateEncodedMultihash (input: string, inputContextForErrorLogging: string) {
```

Comment:
Sanity: maybe I am missing something, not seeing the value of this method if it is just literally calling another method with the exact same params. Remove?

Reply:
Oops